api/src/main/java/com/capitalone/dashboard/config/WebSecurityConfig.java

```diff
@@ -99,6 +109,11 @@ protected StandardLoginRequestFilter standardLoginRequestFilter() throws Excepti
 	protected LdapLoginRequestFilter ldapLoginRequestFilter() throws Exception {
 		return new LdapLoginRequestFilter("/login/ldap", authenticationManager(), authenticationResultHandler);
 	}
+
+	@Bean
+	protected ApiTokenLoginRequestFilter apiTokenLoginRequestFilter() throws Exception {
+		return new ApiTokenLoginRequestFilter("/login/apitoken", authenticationManager(), authenticationResultHandler);
```

Comment:
I don't think you are going to want to piggy back on the ldap and standard login filters.  You don't need to login with an API token, and you aren't going to want a JWT token since you already have an API Token.  You would probably want a filter that is looking at all requests for the API token, and converting that into the Authentication type your provider is looking for.